app/code/Magento/Catalog/Test/Unit/Controller/Category/ViewTest.php

```diff
@@ -5,175 +5,174 @@
+    private $categoryHelperMock;
 
     /**
-     * @var \Magento\Framework\ObjectManagerInterface|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|EventManagerInterface
      */
-    protected $objectManager;
+    private $eventManagerMock;
 
     /**
-     * @var \Magento\Framework\Event\ManagerInterface|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|Layout
      */
-    protected $eventManager;
+    private $layoutMock;
 
     /**
-     * @var \Magento\Framework\View\Layout|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|ProcessorInterface
      */
-    protected $layout;
+    private $processorMock;
 
     /**
-     * @var \Magento\Framework\View\Layout\ProcessorInterface|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|ViewInterface
      */
-    protected $update;
+    private $viewMock;
 
     /**
-     * @var \Magento\Framework\App\ViewInterface|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|Category
      */
-    protected $view;
+    private $categoryMock;
 
     /**
-     * @var \Magento\Backend\App\Action\Context|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|CategoryRepositoryInterface
      */
-    protected $context;
+    private $categoryRepositoryMock;
 
     /**
-     * @var \Magento\Catalog\Model\Category|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|Store
      */
-    protected $category;
+    private $storeMock;
 
     /**
-     * @var \Magento\Catalog\Api\CategoryRepositoryInterface|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|StoreManagerInterface
      */
-    protected $categoryRepository;
+    private $storeManagerMock;
 
     /**
-     * @var \Magento\Store\Model\Store|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|Design
      */
-    protected $store;
+    private $catalogDesignMock;
 
     /**
-     * @var \Magento\Store\Model\StoreManagerInterface|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|ResultFactory
      */
-    protected $storeManager;
+    private $resultFactoryMock;
 
     /**
-     * @var \Magento\Catalog\Model\Design|\PHPUnit_Framework_MockObject_MockObject
+     * @var MockObject|Page
      */
-    protected $catalogDesign;
+    private $pageMock;
 
     /**
-     * @var \Magento\Catalog\Controller\Category\View
+     * @var MockObject|Config
      */
-    protected $action;
+    private $pageConfigMock;
 
     /**
-     * @var \Magento\Framework\Controller\ResultFactory|\PHPUnit_Framework_MockObject_MockObject
+     * @var View
      */
-    protected $resultFactory;
-
-    /**
-     * @var \Magento\Framework\View\Page|\PHPUnit_Framework_MockObject_MockObject
-     */
-    protected $page;
-
-    /**
-     * @var \Magento\Framework\View\Page\Config
-     */
-    protected $pageConfig;
+    private $action;
 
     /**
      * Set up instances and mock objects
      */
     protected function setUp()
```

Comment:
Could you add return type?
```suggestion
    protected function setUp(): void
```